### PR TITLE
resf task 4767: handle event click button next

### PR DESCRIPTION
**Pull request by thanh01684265170**
link: https://edu-redmine.framgia.vn/issues/4767

### CODE REVIEW

**Comment by thanh01684265170** [comment]
ready

**Comment by thanh01684265170** [comment]
ready

**Comment by thanh01684265170** [comment]
ready

File: app/src/main/java/com/framgia/toeic/data/repository/VocabularyLessonRepository.java [hunk 1]
```diff
@@ -22,7 +25,7 @@ public static VocabularyLessonRepository getInstance(VocabularyLessonDataSource.
     }
 
     @Override
-    public void getVocabularies(Callback<List<VocabularyLesson>> Callback) {
-        mVocabularyLessonDataSource.getVocabularies(Callback);
+    public void getLessonVocabularies(Callback<List<VocabularyLessonItem>> Callback) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/toeic/data/repository/VocabularyLessonRepository.java]:
```suggestion
    public void getLessonVocabularies(Callback<List<VocabularyLessonItem>> callback) {
```

File: app/src/main/java/com/framgia/toeic/data/source/local/VocabularyLessonDatabaseHelper.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        mDBHelper = DBHelper;
+    }
+
+    public void getLessonVocabularies(Callback<List<VocabularyLessonItem>> callback) {
+        try {
+            mDBHelper.createDataBase();
+        } catch (IOException e) {
+            callback.onGetDataFail(e);
+        }
+        final List<VocabularyLessonItem> vocabularyLessonItems = new ArrayList<>();
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/toeic/data/source/local/VocabularyLessonDatabaseHelper.java]: remove final

File: app/src/main/java/com/framgia/toeic/data/source/local/VocabularyLessonDatabaseHelper.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        cursorLesson.moveToFirst();
+        do {
+            final VocabularyLessonItem vocabularyLessonItem;
+            vocabularyLessonItem = new VocabularyLessonItem(cursorLesson.getInt(
+                    cursorLesson.getColumnIndex(COLUMN_ID_LESSON)),
+                    cursorLesson.getString(cursorLesson.getColumnIndex(COLUMN_NAME)), false);
+            setVocabularies(vocabularyLessonItem);
+            vocabularyLessonItems.add(vocabularyLessonItem);
+        } while (cursorLesson.moveToNext());
+        callback.onGetDataSuccess(vocabularyLessonItems);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/toeic/data/source/local/VocabularyLessonDatabaseHelper.java]: missing close database

**Comment by thanh01684265170** [comment]
ready

**Pull request merged**